Library/Config.ts

```diff
@@ -71,6 +72,10 @@ class Config {
             : setupString; // CS was invalid, so it must be an ikey
 
         this.instrumentationKey = csCode.instrumentationkey || iKeyCode /* === instrumentationKey */ || csEnv.instrumentationkey || Config._getInstrumentationKey();
+        // validate ikey. If fails throw a warning
+        if(!Config._validateInstrumentationKey(this.instrumentationKey)) {
+            Logging.warn("instrumentation key validation failed");
```

Comment:
Include `this.instrumentionKey` as parameter to `Logging.warn`

Reply:
fixed